cmd/snap-update-ns/common.go

```diff
@@ -79,7 +80,7 @@ func (upCtx *CommonProfileUpdateContext) Lock() (func(), error) {
 	// introduce a symlink that would cause us to mount something other
 	// than what we expected).
 	logger.Debugf("freezing processes of snap %q", instanceName)
-	if err := cgroup.FreezeSnapProcesses(instanceName); err != nil {
+	if err := cgroup.FreezeSnapProcesses(context.Background(), instanceName); err != nil {
```

Comment:
I wonder if we should bubble up this all the way to the top. Ideally we'd cancell the context with `signal.NotifyContext`. Perhaps leave a TODO or even use `context.TODO` to make it clear why.

Reply:
Good point, Added a TODO